### PR TITLE
remove the term "active path" as it was not well defined

### LINKED ISSUE #409: What is an active path?
What is the definition of "active path". It appears to have the meaning of an established path. I wonder if active path and "available path" is to close, and maybe "established path" would make more sense for where "active path" is currently used?

### PR DESCRIPTION
**Pull request by mirjak**
fixes #409

"active path" was used for both: a path that is open or actively used. This PR removed the word active and uses either "open" or "usable" to distinguish these cases.

If more appropriate it might also use the term "established" for an open path. Is that okay, or should we only use "open"?

### CODE REVIEW

File: draft-ietf-quic-multipath.md [hunk 1]
```diff
@@ -447,7 +448,7 @@ the peer to use its own logic to split traffic among available paths.
 
 PATH_BACKUP marks a path as "standby", i.e., it suggests that no traffic
 should be sent on that path if another path is available.
-If all active paths are marked as "standby", no guidance is provided about
+If all established paths are marked as "standby", no guidance is provided about
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:451]: use "open" here as well?

File: draft-ietf-quic-multipath.md [hunk 2]
```diff
@@ -1363,8 +1365,8 @@ followed to limit the amplification risk.
 
 However, while {{QUIC-TRANSPORT}} only allows the use of one path simultaneously
 and therefore only one path migration at the time should be validated,
-this extension allows for multiple open paths, that could in theory be migrated
-all at the same time, and it allows for multiple active paths that could be initialised
+this extension allows for multiple establihed paths, that could in theory be migrated
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:1368]:
```suggestion
this extension allows for multiple open paths, that could in theory be migrated
```

**Review by huitema — APPROVED** [review]

File: draft-ietf-quic-multipath.md [hunk 3]
```diff
@@ -355,7 +356,7 @@ This proposal adds four multipath control frames for path management:
 (see {{path-abandon-frame}}),
 - PATH_BACKUP and PATH_AVAILABLE frames to express a preference
 in path usage (see {{path-backup-frame}} and {{path-available-frame}}), and
-- MAX_PATH_ID frame for increasing the limit of active paths.
+- MAX_PATH_ID frame for increasing the limit of open paths.
```
Comment (qdeconinck) [inline draft-ietf-quic-multipath.md:359]: Just here that I think the "open" word is a bit misleading (but original text was probably)
```suggestion
- MAX_PATH_ID frame for increasing the limit of usable path identifiers.
```

File: draft-ietf-quic-multipath.md [hunk 4]
```diff
@@ -360,7 +361,7 @@ This proposal adds five multipath control frames for path management:
 - PATH_BACKUP and PATH_AVAILABLE frames to express a preference
 in path usage (see {{path-backup-frame}} and {{path-available-frame}}), and
 - MAX_PATH_ID frame (see {{max-paths-frame}}) for increasing the limit of
-active paths, and PATHS_BLOCKED frame (see {{paths-blocked-frame}})
+open paths, and PATHS_BLOCKED frame (see {{paths-blocked-frame}})
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:364]:
```suggestion
path identifiers, and PATHS_BLOCKED frame (see {{paths-blocked-frame}})
```

**Review by qdeconinck — APPROVED** [review]

File: draft-ietf-quic-multipath.md [hunk 5]
```diff
@@ -979,8 +980,9 @@ on another path without further frame scheduling adaptations.
 
 The QUIC specification defines an optional keep alive process, see {{Section 5.3 of QUIC-TRANSPORT}}.
 Implementations of the multipath extension should map this keep alive process to a number of paths.
-Some applications may wish to ensure that one path remains active, while others could prefer to have
-two or more active paths during the connection lifetime. Different applications will likely require different strategies.
+Some applications may wish to ensure that one only path remains usable, while others could prefer to maintain
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:983]:
```suggestion
Some applications may keep only one path alive, while others could prefer to maintain
```

**Pull request merged**